packages/@okta/courage-for-signin-widget/webpack.config.js

```diff
@@ -3,7 +3,6 @@ const { resolve } = require('path');
 const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const { BannerPlugin } = require('webpack');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const ReplaceInFileWebpackPlugin = require('replace-in-file-webpack-plugin');
```

Comment:
What is the reason to remove this ?

Reply:
We used this dependency as a workaround for removing a deprecated API from the Courage bundle. Now that Courage uses the updated API - we can safely remove it 😄 

See [this comment](https://github.com/okta/okta-signin-widget/pull/2048/files#diff-eec33d248efcfb44568c1072566a82e05989edd169d3635238f2c1ee52e6ad9dL121-L123) for more details.